release_notes/v2_vs_v3.adoc

```diff
@@ -0,0 +1,107 @@
+= Comparing OpenShift version 2 and OpenShift version 3
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: This topic is a list of the differences between OpenShift v2 and OpenShift v3.
+
+toc::[]
+
+== Overview
+While still an application platform, OpenShift v3 is a very different product than OpenShift v2. Many of the same terms are used, and the same functions are performed, but the terminology can be different, and behind the scenes things may be happening very differently.
+
+This topic discusses these differences in detail, in an effort to help OpenShift users in the transition from OpenShift v2 to OpenShift v3.
+
+== Architecture Changes
+
+*Gears vs Containers*
+
+Gears were a core component of OpenShift v2. Technologies such as kernel namespaces, cGroups, and SELinux helped deliver a highly-scalable, secure, containerized application platform to OpenShift users. Gears themselves were a form of container technology.
+
+OpenShift v3 takes the gears idea to the next level. It uses Docker as the next evolution of the v2 container technology. This container architecture is at the core of OpenShift v3.
+
+*Kubernetes*
+
+As applications in OpenShift v2 typically spanned over multiple gears, applications on OpenShift v3 will expectedly span over multiple containers. In OpenShift v2, gear orchestration, scheduling, and placement was handled by the OpenShift broker host. OpenShift v3 integrates Kubernetes into the master host to drive container orchestration. 
```

Comment:
Hmm, "spans over STUFF" sounds strange to me.  I'm used to hearing simply "spans STUFF" (no preposition).